neo/NeoSystem.cs

```diff
@@ -13,13 +13,7 @@ namespace Neo
 {
     public class NeoSystem : IDisposable
     {
-        public ActorSystem ActorSystem { get; } = ActorSystem.Create(nameof(NeoSystem),
```

Comment:
@erikzhang  but what is this? This is present in our master branch

Reply:
https://github.com/neo-project/neo/blob/0a85a9eba2bc17a07d41f79d8cfe0276156ba6d0/neo/NeoSystem.cs#L16

Reply:
`=` is an assigning operator. It will be run only once.

Reply:
Oh, now I get it. I didn't know that this runs the code only once.
But what about the initialization order?  It is not explicit, and there is evidence that this is causing an error

Reply:
From @jsolman :

> I have heard reports of the plugins not running consistently for the first few blocks upon starting the process. This should address that issue. The plugins aren’t creating actors in their constructors, so this should be compatible and ready for testing.

Reply:
Its order is deterministic and will run before the constructor. It has not been confirmed that it will cause problems.

Reply:
Is there a reason we don't call it in the first line of the constructor?